vulnerability/events_creator.go

```diff
@@ -171,9 +173,8 @@ func (e EventsCreator) generateEvent(reportResult trivy_types.Result, vul trivy_
 					Vendor:  vulScannerVendor,
 				},
 				Score: Score{
-					Impact:  cvssScore,
```

Comment:
Since we are removing `Impact`.
We need to later on update our technical index to make sure that it synced.